### ISSUE TITLE
Failing Hexagon registration

### ISSUE THREAD

**Issue by wthaem** [mitxel-m/gdal-ecw-plugin#1]
@mitxel-m: Could you describe the details of HEXAGON registration? I have got a personal Hexagon login. But without a valid company and/or administrator I don't get access to a ECW SDK license and to the download of the package. 
Thanks!

**Comment by mitxel-m** [comment]
@wthaem When I downloaded the SDK some time ago the registration was as easy as filling in a form and you had access to the download page.

I have checked today and I see that it is no longer like that. I am surprised with the changes in the Hexagon website, and I am in the same case as you, even if I can login with my old username/password I can not access the download page to get the sdk. I have filled out a new form, as indicated on their website, but I have no response yet.

I will give it some time, to see if there is a response, before asking Hexagon. Searching the net hasn't helped either.

Fortunately I still have a copy of the SDK for "personal" use if I need to upgrade my system.

**Comment by wthaem** [comment]
Thanks for your attempts to help!

> I have filled out a new form, as indicated on their website, but I have no response yet.

The same here. Even trying to contact them as indicated on their webpage failed due to the missing valid company name. 

For the moment being I use the gisinternals package with ECW support for Windows to convert the ECW files to TIFF files and use these converted files in QMS.

**Comment by mitxel-m** [comment]
Although it is not obvious on the new Hexagon website, I have managed to get it recently as follows:

1. On the [Hexagon download web page](https://download.hexagongeospatial.com), click on " Community Access Request"
2. Fill in the form, and in the comment box  write a request for the ECW-SDK. 
3. The next day I received an email with a temporary link to download it.

@wthaem  regarding getting support for ECW in QMS 1.16.1 for windows: you can add to QMS  the GISinternals  plugin matching  GDAL version 3.4.1 ([info updated here](https://github.com/Maproom/qmapshack/wiki/DocMapsTipsRasterDEM#add-ecw-plugin)) . Then  you can use ECW with no need to convert to TIFF.

**Comment by wthaem** [comment]
@mitxel-m : 

> The next day I received an email with a temporary link to download it.

Thanks for the hint. Succeeded to get the link after the nth attempt.

> regarding getting support for ECW in QMS 1.16.1 for windows:

Can confirm this. My environment is a personal build and is slighly different from the official QMS version:
* Latest QMS dev version
* VisualStudio 2022 with generator "Visual Studio 16 2019"  (x64)
* GDAL version 3.6.1: this one only supports the new `cmake` build procedure. `nmake` build is not supported anymore. I am still trying to find the correct build parameters for the GDAL plugins.

QMS itself is ok. MBTiles and MrSID can already be used in QMS, ECW failed up to now, even when trying to use the gisinternal variant you described.

 I'll report as soon as I had some success.

**Comment by wthaem** [comment]
@mitxel-m, @kiozen: (just want to fulfil my promise to report) My attempts to build QMS/QMT with GDAL 3.6.2 on Windows failed.

* I succeeded to build GDAL with the new cmake build procedure (including the MrSID and ECW plugins). 
* The ECW plugin is supported and works fine only if the environment variable `GDAL_DRIVER_PATH` is properly set (as recommended in the GDAL docu). 
* In the QMS/QMT `CAppSetupWin.cpp` file this variable is unset by an unknown reason (ECW works fine after removing the unset line!). Therefore, my GDAL build can't be used in QMS/QMT.

I don't have any idea how to proceed. Best would be to get GDAL running without this environment variable but here I failed.
